pkg/spec/gitops.go

```diff
@@ -526,7 +525,11 @@ func parseQueries(top map[string]json.RawMessage, result *GitOps, baseDir string
 	return multiError
 }
 
-func parseSoftware(softwareRaw json.RawMessage, result *GitOps, baseDir string, multiError *multierror.Error) *multierror.Error {
+func parseSoftware(top map[string]json.RawMessage, result *GitOps, baseDir string, multiError *multierror.Error) *multierror.Error {
```

Comment:
Is the following syntax supported?
```
software:
  - path: ../lib/eng_software.yml
  - path: ../lib/mgr_software.yml
```

If not, can you file a bug? This is how other top level keys work.

Reply:
i ran a quick test and that's still working